bundles/org.openhab.binding.siemenshvac/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon xmlns:addon="https://openhab.org/schemas/addon/v1.0.0" id="siemenshvac"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<type>binding</type>
+	<name>SiemensHvac Binding</name>
+	<description>This is the binding for SiemensHvac.</description>
+	<connection>local</connection>
+	<discovery-methods>
+		<discovery-method>
+			<service-type>upnp</service-type>
+			<match-properties>
+				<match-property>
+					<name>modelName</name>
+					<regex>Web Server OZW.*</regex>
+				</match-property>
+			</match-properties>
```

Comment:
I almost forgot: Is there something else that could also be matched, for example manufacturer?

Reply:
the remotedevice look like this:

![image](https://github.com/openhab/openhab-addons/assets/4678815/999fb21d-9cce-4ce7-977c-01c94a28154c)

we potentially could add a match on the manufacturer : "Siemens Switzerland Ltd."
Or perhaps more simply to be sure: 

<match-property>
                    <name>manufacturer</name>
                    <regex>Siemens.*</regex>
</match-property>

Reply:
Agreed, it should be sufficient with `Siemens.*` like I see you did. 👍